saleor/payment/gateways/adyen/tests/webhooks/test_handle_notifications.py

```diff
@@ -22,6 +23,8 @@
     webhook_not_implemented,
 )
 
+logger = logging.getLogger(__name__)
+
```

Comment:
What do you think about adding a test only for `get_payment`?  It could use the `.parametrize` and we could test different types of ID, like numbers, null, str etc. Just to be sure that we handle all standard scenario